main.cpp

```diff
@@ -1010,7 +901,375 @@ Part 1e - Step 19: Request a review
+    3) maximum pressure in bar (float)
+    4) colour (std::string)
+    5) percentage of wear (float)
+3 things it can do:
+    1) loose pressure
+    2) bounce
+    3) spin
+*/
+struct Wheel
+{
+    //5 properties:
+    //    1) material (std::string)
+    std::string material = "rubber";
+    //    2) wheel rim type (std::string)
+    std::string rimType = "aluminum";
+    //    3) maximum pressure in bar (float)
+    float maxBarPressure = 6.2f;
+    //    4) colour (std::string)
+    std::string colour = "black";
+    //    5) percentage of wear (float)
+    float wearPercentage = 0.0f;
+    //3 things it can do:
+    //    1) loose pressure
+    void loosePressure(float pressureAmount); // loose pressure of a given pressure amount
+    //    2) bounce
+    void bounce();
+    //    3) spin
+    void spin(int revolutionsPerMinute); // set the revolution per minute to make the wheel spinning
+};
+
+/*
+Thing 7) Seat
+5 properties:
+    1) material (std::string)
+    2) shape (std::string)
+    3) number (int)
+    4) colour (std::string)
+    5) percentage of cushioning (float)
+3 things it can do:
+    1) absorb vibrations
+    2) cover objects holder
+    3) comfort driver
+*/
+struct Seat
+{
+    //5 properties:
+    //    1) material (std::string)
+    std::string material = "leather";
+    //    2) shape (std::string)
+    std::string shape = "rounded";
+    //    3) number (int)
+    int numberOfSeats = 1;
+    //    4) colour (std::string)
+    std::string colour = "black";
+    //    5) percentage of cushioning (float)
+    float cushioningPercentage = 90.0f;
+    //3 things it can do:
+    //    1) absorb vibrations
+    void absorbVibrations();
+    //    2) cover objects holder
+    void coverObjectsHolder();
+    //    3) comfort driver   
+    void comfortDriver();
+};
+
+/*
+Thing 8) Light
+5 properties:
+    1) wattage (float)
+    2) brightness percentage (double)
+    3) colour (std::string)
+    4) light anglulation in degrees (int)
+    5) type of light (std::string)
+3 things it can do:
+    1) illuminate street
+    2) alert people
+    3) indicate change of direction
+*/
+struct Light
+{
+    //5 properties:
+    //    1) wattage (float)
+    float wattage = 60.0f;
+    //    2) brightness percentage (double)
+    double brightnessPercentage = 80.5;
+    //    3) colour (std::string)
+    std::string colour = "white";
+    //    4) light anglulation in degrees (int)
+    int degreesAngulation = 90;
+    //    5) type of light (std::string)
+    std::string type = "halogen";
+    //3 things it can do:
+    //    1) illuminate street
+    double illuminateStreet(double brightnessAmount); // Based upon the brightness amount, it returns the updated brightness percentage
+    //    2) alert people
+    void alertPeople();
+    //    3) indicate change of direction
+    std::string changeDirection(int direction); // direction could be 1 or 2 to indicate left or right. It returns the direction as a string.
```

Comment:
the action is 'Indicate", not "Change"
the current function name is therefore incorrect.
- [ ] revise